### PR TITLE
Upgrade config file

### PR DESCRIPTION
**Pull request by ericgaspar**
Patches that allows to connect to the app without authentication are not working anymore.
I see 3 possible solutions:
- rework patches to work with upstream changes
- use built-in authentication
- set authentication to false and protect app access with SSO (as in this PR)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -89,21 +95,10 @@ ynh_system_user_create --username=$app --home_dir=$final_path
...
-	ynh_setup_source --dest_dir="$tmpdir"
-
-	# Backup the config file in the temp dir
-	cp -a "$final_path/config.php" "$tmpdir/config.php"
-
-	# Replace the old ttrss by the new one
-	ynh_secure_remove --file="$final_path"
-	mv "$tmpdir" "$final_path"
-	ynh_secure_remove --file="$tmpdir"
+	ynh_setup_source --dest_dir="$final_path" #--keep="$final_path/config.php"
```
Comment (tituspijean) [inline scripts/upgrade:101]: Should the `--keep` be actually removed?

Comment (ericgaspar) [reply scripts/upgrade]: indeed! the config file syntax as completely changed

**Review by tituspijean — APPROVED** [review]

**Pull request merged**